python/pylibcudf/pylibcudf/io/csv.pyi

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2024, NVIDIA CORPORATION.
 
 from collections.abc import Mapping
+from typing import Self
```

Comment:
Since cudf still supports Python 3.10, we'll need to import this from `typing_extensions`

Reply:
Thanks, done